src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -57,7 +57,7 @@
 
         <service id="validator.expression" class="%validator.expression.class%">
             <argument type="service" id="property_accessor" />
-            <tag name="validator.constraint_validator" alias="validator.expression" />
+            <tag name="validator.constraint_validator" alias="Symfony\Component\Validator\Constraints\ExpressionValidator" />
```

Comment:
the old alias should still be supported to keep support for older versions of the component